src/helpers/editing-db.handler.ts

```diff
@@ -66,12 +66,20 @@ export const editTeamMember = async (team: string, person: APIPerson): Promise<v
 export const deleteTeamMember = async (person: APIPerson): Promise<void> => {
 
   const { team, lp, ...mappedPerson } = person;
-
   axios.patch(`/registry/${team}/${person.id}.json`, mappedPerson); 
 
-  store.dispatch(reduxDeleteMember(person));
+  // store.dispatch(reduxDeleteMember(person));
+  team && store.dispatch(reduxEditMember(person, team));
 };
 
+export const permDeleteTeamMember = async (person: APIPerson): Promise<void> => {
```

Comment:
We can use long names, if them understanding is better :) in this case we can use permament instead of perm :)

Reply:
done